scene/3d/visual_instance_3d.cpp

```diff
@@ -357,17 +440,20 @@ void GeometryInstance3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_lod_bias", "bias"), &GeometryInstance3D::set_lod_bias);
 	ClassDB::bind_method(D_METHOD("get_lod_bias"), &GeometryInstance3D::get_lod_bias);
 
-	ClassDB::bind_method(D_METHOD("set_lod_max_hysteresis", "mode"), &GeometryInstance3D::set_lod_max_hysteresis);
-	ClassDB::bind_method(D_METHOD("get_lod_max_hysteresis"), &GeometryInstance3D::get_lod_max_hysteresis);
+	ClassDB::bind_method(D_METHOD("set_visibility_range_end_hysteresis", "distance"), &GeometryInstance3D::set_visibility_range_end_hysteresis);
+	ClassDB::bind_method(D_METHOD("get_visibility_range_end_hysteresis"), &GeometryInstance3D::get_visibility_range_end_hysteresis);
+
+	ClassDB::bind_method(D_METHOD("set_visibility_range_end", "distance"), &GeometryInstance3D::set_visibility_range_end);
+	ClassDB::bind_method(D_METHOD("get_visibility_range_end"), &GeometryInstance3D::get_visibility_range_end);
 
-	ClassDB::bind_method(D_METHOD("set_lod_max_distance", "mode"), &GeometryInstance3D::set_lod_max_distance);
-	ClassDB::bind_method(D_METHOD("get_lod_max_distance"), &GeometryInstance3D::get_lod_max_distance);
+	ClassDB::bind_method(D_METHOD("set_visibility_range_begin_hysteresis", "distance"), &GeometryInstance3D::set_visibility_range_begin_hysteresis);
+	ClassDB::bind_method(D_METHOD("get_visibility_range_begin_hysteresis"), &GeometryInstance3D::get_visibility_range_begin_hysteresis);
 
-	ClassDB::bind_method(D_METHOD("set_lod_min_hysteresis", "mode"), &GeometryInstance3D::set_lod_min_hysteresis);
-	ClassDB::bind_method(D_METHOD("get_lod_min_hysteresis"), &GeometryInstance3D::get_lod_min_hysteresis);
+	ClassDB::bind_method(D_METHOD("set_visibility_range_begin", "distance"), &GeometryInstance3D::set_visibility_range_begin);
+	ClassDB::bind_method(D_METHOD("get_visibility_range_begin"), &GeometryInstance3D::get_visibility_range_begin);
 
-	ClassDB::bind_method(D_METHOD("set_lod_min_distance", "mode"), &GeometryInstance3D::set_lod_min_distance);
-	ClassDB::bind_method(D_METHOD("get_lod_min_distance"), &GeometryInstance3D::get_lod_min_distance);
+	ClassDB::bind_method(D_METHOD("set_visibility_range_parent", "parent"), &GeometryInstance3D::set_visibility_range_parent);
```

Comment:
I would most likely do this internally, and not expose the parent range or clearing parent to the user. As in, user should only need to be able to set the children.

Reply:
What do you mean with "set the children"? Exposing a method that receives a list of nodes and then sets their parent internally?

The issue with that is it hides the fact that nodes can only have one parent, if properly documented it won't be that bad, but I'm still not sure what are the benefits of setting the children instead of setting the parent...

I can see why it's needed in the editor, where you just edit one node and select all of its children, but from the scripting side it's just a matter of iterating your list of children and setting their parent node.

Reply:
Fixed in the latest version of the PR.